OasisDEX/oasis-borrow
# sidebar form add atp trigger

# [Auto Take Profit form](https://app.shortcut.com/oazo-apps/story/5542/ui-auto-take-profit-set-up-form-parameters)

<please insert a shortcut link above>
  
## Changes 👷‍♀️
![image](https://user-images.githubusercontent.com/6871923/193038136-0e940003-421f-4511-a76e-b8a835b63513.png)
  
## How to test 🧪
- Check if form matches figma designs (note final min and max for slider for various colllaterals is separate story)
- check if all acceptance criteria are met
- check if there are no unexpected errors
